arches/app/media/js/views/components/search/advanced-search.js

```diff
@@ -45,6 +49,55 @@ define([
                 self.filter.facets.remove(facet);
             };
 
+            var aliasToNodeName = function(path, pathData, graphid) {
+
+                var lookupName = function(alias) {
+                    var namePath = pathData[graphid].filter((cardData) => 
+                    cardData[3] == alias
+                    );
+                    return namePath[0][4];
+                }
+
+                self.cardPath(path.map((alias) => 
+                    lookupName(alias)
+                    )
+                );
+            }
+
+            this.requestPathData = function(graph_id, nodegroup_id) {
+                $.ajax({
+                    type: "GET",
+                    url: arches.urls.api_get_nodegroup_tree, 
+                    data: {graphid: graph_id},
+                    context: this
+                }).done(function(response) {
+                    pathData[graph_id] = response['path'];
+                    var graphPath = pathData[graph_id].filter((d) => d[1] === nodegroup_id)
+                    aliasToNodeName(graphPath[0][6].split(' - '), pathData, graph_id)  
+               })
+            };
+
+            this.expandPath = ko.pureComputed(function() {
+                return self.selectedNodegroupId();
+            }); 
+
+            this.getNodegroupPath = function() {
+                if (self.selectedNodegroupId() === this.nodegroup_id) {
+                    self.selectedNodegroupId(undefined);
+                }
+                else if (self.selectedNodegroupId() != this.nodegroup_id) {
```

Comment:
not a big deal, I just suggest strict inequality (!==) as the loose inequality (!=) signals to me as a code reader that you wanted type conversion.
```suggestion
                else {
```